### PR TITLE
nk3: Add system information to test log

### PR DESCRIPTION
**Pull request by robin-nitrokey**
This patch adds information about the host system to the log for the nk3
test subcommand.

### CODE REVIEW

File: pynitrokey/cli/nk3/test.py [hunk 1]
```diff
@@ -76,6 +77,11 @@ def log_devices() -> None:
         logger.info(f"- {descriptor.path} ({descriptor.vid:x}:{descriptor.pid:x})")
 
 
+def log_system() -> None:
+    logger.info(f"platform: {platform.platform()}")
+    logger.info(f"uname: {platform.uname()}")
+
```
Comment (szszszsz) [inline pynitrokey/cli/nk3/test.py:83]: Ideally for Linux I would log the distribution as well, though it seems to not be available in the standard library unfortunately. 
There is `distro` package. We could include `/etc/os-release` as-is to the logs as well, without any parsing or library. Next iteration please.

**Review by szszszsz — APPROVED** [review]

File: pynitrokey/cli/nk3/test.py (hunk 1 above)
Comment (robin-nitrokey) [reply pynitrokey/cli/nk3/test.py:83]: There also is [`platform.freedesktop_os_release()`](https://docs.python.org/3/library/platform.html#platform.freedesktop_os_release) but it requires Python 3.10.

For Debian, `uname` contains the distribution.  Is that not true for other distributions?

Comment (szszszsz) [reply pynitrokey/cli/nk3/test.py:83]: That's my output:
```
$ platform.uname()
uname_result(system='Linux', node='stumpy', release='5.13.19-100.fc33.x86_64', version='#1 SMP Sat Sep 18 16:33:11 UTC 2021', machine='x86_64')
```

Comment (robin-nitrokey) [reply pynitrokey/cli/nk3/test.py:83]: That’s unfortunate.  For comparison, mine is `uname_result(system='Linux', node='nitropc', release='5.10.0-9-amd64', version='#1 SMP Debian 5.10.70-1 (2021-09-30)', machine='x86_64')`.

Comment (szszszsz) [reply pynitrokey/cli/nk3/test.py:83]: It will do

**Pull request merged**